carbon-design-system/carbon
# chore(Tabs): added prop deprecation warnings for v11

REF #9535

#### Changelog

- Added deprecation warnings for handleTabClick and handleTabKeyDown to `Tab`.
- Updated the PublicAPI test snapshot.
- Added comments in the appropriate areas to remove deprecations for the next major release.

#### Testing / Reviewing

- Check the `Tab` stories and make sure everything is firing off as it should.
- Make sure tests are passing.
